packages/atlas/src/themesource/atlas_core/web/_exclusion-variables.scss

```diff
@@ -60,6 +63,8 @@ $exclude-range-slider: false;
 $exclude-range-slider-helpers: false;
 $exclude-rating: false;
 $exclude-rating-helpers: false;
+$exclude-scroll-container: true;
```

Comment:
do we still need this value? shouldn't `$use-modern-client` is sufficient?
and why 2 variables (`exclude-scroll-container` + `exclude-scroll-container-dojo`) if putting both `true` is not possible?

Reply:
Indeed, good point, I reduced it to one as it was before and check `$use-modern-client` under the hood.